notes.md

```diff
@@ -0,0 +1,1909 @@
+>>> a = torch.tensor(10)
+>>> a.T
+>>> a.H
+```
+
+</td>
+<td>
+
+```Python
+>>> a = torch.tensor(10)
+>>> a.T
+UserWarning: Tensor.T is deprecated on 0-D tensors. This function is the identity in these cases.
+>>> a.H
+UserWarning: Tensor.H is deprecated on 0-D tensors. Consider using x.conj().
+```
+
+</td>
+</tr>
+</table>
+
+## Autograd API
+
+### **Deprecate decorating classes with torch.no_grad (#89522)**
+
+Decorating classes with `torch.no_grad` is now deprecated. You should be decorating its functions or methods instead. To preserve the current behavior of class decoration, you can directly decorate the `__init__` method and nothing else.
+
+<table>
+<tr>
+<th>1.13</th>
+<th>2.0</th>
+</tr>
+<tr>
+<td>
+
+```Python
+@torch.no_grad()
+class Blah():
+  pass
+```
+
+</td>
+<td>
+
+```Python
+class Blah():
+  @torch.no_grad()
+  def __init__(self):
+    pass
+```
+
+</td>
+</tr>
+</table>
+
+## Linalg
+
+### **Remove the use of overload at::frobenius_norm(const Tensor&) (#81762)**
+
+In continuation with the deprecation process from release 1.12 the tensor overload for this function has been removed. This function was not used in the bindings of Pytorch and should not impact users of `torch.norm`.
+
+## torch.nn API
+
+### **Canceling deprecation of `functional.{tanh, sigmoid}` functions (#86905)**
+
+Both these ops are heavily used and so will not be removed. Deprecation warnings have been removed.
+
+### **Deprecated torch.nn.utils.stateless.functional_call in favor of torch.func.functional_call (#92280)**
+
+We’ve moved torch.nn.stateless.functional_call under the torch.func module to reflect how it is useful for working with nn.Modules in a functional style. As of PyTorch **2.0**, `torch.func.functional_call` is a drop-in replacement for `torch.nn.stateless.functional_call` and we will remove `torch.nn.utils.stateless.functional_call` in a future version of PyTorch. However, please note that we did change the default behavior of `torch.nn.stateless.functional_call` in PyTorch 2.0 (see “torch.nn.utils.stateless.functional_call now respects tied weights” under BC-breaking notes).
+
+## Releng
+
+### **Deprecated private API torch.\_six (#94709)**
+
+Removed the Python 2 and 3 compatibility library six and future and torch.\_six.
+**2.0**
+
+```Python
+# from torch._six import string_classes
+str
+# from torch._six import int_classes
+int
+# from torch._six import inf, nan
+from torch import inf, nan
+# torch._six.string_classes
+str
+```
+
+## Onnx
+
+### **Deprecated Caffe2 ONNX exporter support[ #95071](https://github.com/pytorch/pytorch/pull/95071)**
+
+Users must use PyTorch 1.x versions to use Caffe2 ONNX exporter. This capability will be completely removed from PyTorch 2.x series.
+
+# New Features
+
+## torch.nn API
+
+- Add `torch.nn.functional.scaled_dot_product_attention()` to allow writing fast Transformer-like functions and use it to speed up `nn.Transformer()` ( #91362, #91066, #90413, #87312, #94008, #89470, #90776, #92189)
```

Comment:
You lost all the PR links in the export :( 
You most likely want a magic replace regexp from #{number} -> [#{number}](github url)

Reply:
Hoooo true! 
The rendering in the PR doesn't pick that up for some reason. Great!